_config.yml

```
@@ -3,7 +3,7 @@
 #
 
 # Name of your site (displayed in the header)
-name: Apoapsis 2019
+name: Apoapsis 2020
```

```suggestion
name: Apoapsis 2020/21
```

Auf der CanSat-Homepage steht ja 2020/21, vielleicht sollten wir das dann auch so machen